python/marvin/contrib/vacs/mangagema.py

```diff
@@ -0,0 +1,66 @@
+# !usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under a 3-clause BSD license.
+#
+# @Author: Brian Cherinka
+# @Date:   2018-10-11 17:51:43
+# @Last modified by:   Brian Cherinka
+# @Last Modified time: 2018-11-29 17:23:15
+
+from __future__ import print_function, division, absolute_import
+
+import marvin.tools
+
+from .base import VACMixIn
+
+
+class GEMAVAC(VACMixIn):
+    """Provides access to the MaNGA-GEMA VAC.
+
+    VAC name: GEMA
+
+    URL: link
+
+    Description: Returns GEMA VAC
+
+    Authors: Maria Argudo-Fernandez, Daniel Goddard, Daniel Thomas, Zheng Zheng, Lihwai Lin, Ting Xiao, Fangting Yuan, Jianhui Lian, et al
+
+    """
+
+    # Required parameters
+    name = 'mangagema'
+    description = 'Returns GEMA table data'
+    version = {'MPL-7': '1_0_1',
+               'DR15': '1_0_1'}
+
+    # optional Marvin Tools to attach your vac to
+    include = (marvin.tools.cube.Cube,
+               marvin.tools.maps.Maps,
+               marvin.tools.modelcube.ModelCube)
+
+    # Required method,
+    def get_data(self, parent_object):
+
+        # get any parameters you need from the parent object
+        release = parent_object.release
+        
+        # define the variables to build a unique path to your VAC file
+        path_params = {'ver': self.version[release]}
+
+        # get_path returns False if the files do not exist locally
+        allfile = self.get_path('mangagema', path_params=path_params)
+
+        # download the vac from the SAS if it does not already exist locally
+        if not allfile:
+            allfile = self.download_vac('mangagema', path_params=path_params)
+
+        return allfile
```

Comment:
`allfile` is just a string path to your VAC file on disk.  If you want to return data, you'll need to open the FITS file.

Reply:
Thanks for the comments!

Can I open the file inside the GEMAVAC class or it would be better to add a new class? 
I think it would be nice to get the data in each table of the VAC, only for the row corresponding to the requested galaxy.

Reply:
I don't think you need a new class since all you want to do is return row data.  You can write a new function that takes as input, the path to the VAC file, `allfile`, and mangaid, and returns a dictionary containing the VAC data for that target.  For example,
```
def get_gema_data(path, mangaid):
    # open the file
    # create a dictionary
    # if the mangaid is not in your VAC return an empty dictionary
    # if it is, then look up the row data for that mangaid for each extension and add it to the dictionary using the name of the extension as dictionary key
    # return the dictionary 
```


Then inside `get_data` , you can call that function and return the data , e.g.
```
gema_data = get_gema_data(allfile, mangaid)
return gema_data 
```

Reply:
Or you can create an Astropy Table for that target and put in it all the data from your VAC for that target.  Or you can create a little mini FITS file with those rows selected out.  Or a Pandas Dataframe.  It's up to you.